### DISCUSSION TITLE
Drag and drop not working in linux verison!

### DISCUSSION THREAD

**Discussion by SiavashMaraghechi** [Ifsttar/I-Simpa#377] [Q&A] [answered]
Hi everyone,

After loads of struggles I managed to install i-Simpa on linux (Ubuntu 22.04.03). All seems to be working fine at early stages of usage except I can't drag and drop anything on the tree of the project window! Can this be fixed? Or better, is there any other to assign, for example, surfaces to a surface receiver? In the documentation I couldn't find any other way mentioned. Thanks for the help :)

**Comment by nicolas-f** [comment]
Hi

It has been fixed on linux_release branch among other things:

https://github.com/Universite-Gustave-Eiffel/I-Simpa/tree/linux_release

**Reply by SiavashMaraghechi** [reply]
Thanks a lot for the response. I've been busy building based on the linux_release branch and am having trouble with boost and python 3.10. I'm heavily using chat gpt's help for all these steps since otherwise it would be impossible for me to manage it all. Hopefully I'll manage so I'll get to my drag and drop as well :)

**Reply by nicolas-f** [reply, marked as answer]
We are using github action to build I-Simpa for linux:

https://github.com/Universite-Gustave-Eiffel/I-Simpa/actions/runs/7567983783

It is based on ubuntu 22-04 as you can see in the workflow file

**Reply by SiavashMaraghechi** [reply]
Thanks a lot this worked fine :)
